src/Http/Router.php

```diff
@@ -56,6 +56,19 @@ function () use ( $route, $methods ) {
 		);
 	}
 
+	private function get_current_route() {
+		$wp_url        = get_site_url();
+		$installed_dir = parse_url( $wp_url, PHP_URL_PATH );
+
+		// requested uri relative to WP install
+		$request_uri = str_replace( $installed_dir, '', $_SERVER['REQUEST_URI'] );
+
+		$uri   = sanitize_text_field( wp_unslash( $request_uri ) );
```

Comment:
Shouldn't this be `sanitize_url()`?

Reply:
Good catch! Fixed https://github.com/Automattic/wp-openid-connect-server/pull/30